File: 01.Fundamental_Level_Java_2016/01.Java_Fundamentals_Mar_2016/02.Homewok/src/TriangleArea.java
import java.util.ArrayList;
import java.util.Scanner;

public class TriangleArea {

    public static void main(String[] args) {

        Scanner scn = new Scanner(System.in);

        ArrayList<Integer> coordinates = new ArrayList<>();

        for (int i = 0; i < 3; i++) {
            String[] points = scn.nextLine().split(" ");
            coordinates.add(Integer.parseInt(points[0]));
            coordinates.add(Integer.parseInt(points[1]));
        }
        int pointAx = coordinates.get(0);
        int pointAy = coordinates.get(1);
        int pointBx = coordinates.get(2);
        int pointBy = coordinates.get(3);
        int pointCx = coordinates.get(4);
        int pointCy = coordinates.get(5);

        double areaInDouble = (pointAx * (pointBy - pointCy) + pointBx * (pointCy - pointAy) + pointCx * (pointAy - pointBy)) / 2;
        areaInDouble = Math.abs(areaInDouble);

        int area = (int)Math.round(areaInDouble);
        System.out.println(area);
    }
}

File: 00.My_Parctice_Courses/05.JavaScript_Fundamentals_Sep_2016/02.JSControlFlowLogic/triangleArea.js
function triangleArea(sides) {
    var a = Number(sides[0]);
    var b = Number(sides[1]);
    var c = Number(sides[2]);

    var s = (a + b + c) / 2;

    var area = Math.sqrt(s * (s - a) * (s - b) * (s - c));

    console.log(area);
}

triangleArea(['2', '3.5', '4']);
